nservicebus/azure-service-bus/configuration/configuration.md

```diff
@@ -0,0 +1,200 @@
+ * `OversizedBrokeredMessageHandler<T>(T)`: This setting allows to override the transports behavior when a single message exceeds the maximum message size. The default behavior is represented by `ThrowOnOversizedBrokeredMessages` and will throw a `MessageTooLargeException`, informing to consider using the Databus feature instead.
+
+## Transactional behavior
+
+Under certain conditions, the Azure Service Bus transport can guarantee transactional behavior by combining the send operations and completion of the receive operation in one atomic operation. These conditions include that the source and destination are in the same namespace, that messages are received using `ReceiveMode.PeekLock` and that the transport explicitly sends messages via the receive queue. The latter can be configured using:
+
+  * `SendViaReceiveQueue(bool)`: Uses the receive queue to dispatch outgoing messages when possible. Defaults to true.
+
+## Physical Addressing Logic
+
+Next to managing connectivity and the layout of messaging entities, another important task of the transport is to figure out how these entities are named and where they can actually be found in the (seemingly infinite) cloud. This is referred to as the 'Physical Addressing Logic', and has the responsibility to turn logical endpoint names into physical addresses of Azure Service Bus entities. The following configuration sections allow to redefine this aspect of the transport.
+
+ * `UseNamespaceNamesInsteadOfConnectionStrings()`: Causes the transport to pass around namespace names instead of raw connection strings in brokered message body headers.
+ * `Validation()`: Provides access to the settings that determine how entity names are validated.
+ * `Sanitization()`: Provides access to the settings that determine how invalid entity names are cleaned up to become valid.
+ * `Individualization()`: Provides access to the settings that determine how entity names are modified when usedin different consumption models.
+ * `NamespacePartitioning()`: Provides access to the settings that determine how entities are partitioned across namespaces.
+ * `Composition()`: Provides access to the settings that determine how entities are composed inside a single namespace.
+ 
+### Validation
+
+These settings determine how entity names are validated. These settings should correspond to the actual validation rules in place in the Azure Service Bus namespace configured. Different types of namespaces have different implementations and these implementations have changed over time as well (without notice and sometimes confusing/outdated msdn documentation). The default settings align with recently created Standard namespaces.
```

Comment:
How about

The validation settings determine how entity names are validated. They should correspond to the actual validation rules in the configured Azure Service Bus namespace. The rules implementations vary depending on the namespace type, and have changes over time (in some cases without notice and update of the relevant MSDN documentation). The default settings align with recently created Standard namespaces.